### PR TITLE
Migrate docs from GitBook

### PR DESCRIPTION
**Pull request by cwarnermm**
This PR migrates the Confluence plugin documentation https://mattermost.gitbook.io/plugin-confluence/ currently hosted via GitBook to the GitHub repository README file.

Once this PR is merged, the GitBook-hosted content can be archived/removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,137 @@
...
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-confluence/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-confluence/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-A Mattermost plugin for Confluence. Supports Confluence Cloud, Server and Data Center versions. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated.  Comments and activity can be pushed into specific Mattermost channels for full visibility. 
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, and environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plguin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in.
```
Comment (catalintomai) [inline README.md:10]: plguin->plugin

File: README.md [hunk 2]
```diff
@@ -5,13 +5,137 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-confluence)](https://github.com/mattermost/mattermost-plugin-confluence/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-confluence/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-confluence/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-A Mattermost plugin for Confluence. Supports Confluence Cloud, Server and Data Center versions. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated.  Comments and activity can be pushed into specific Mattermost channels for full visibility. 
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, and environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility.
```
Comment (catalintomai) [inline README.md:8]: remove extra "and"?

File: README.md [hunk 3]
```diff
@@ -5,13 +5,137 @@
...
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, and environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plguin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in. 
 
-Installation and Usage instructions are located here: https://mattermost.gitbook.io/plugin-confluence/
+- In a Mattermost channel, setup subscriptions to get updates from Confluence and manage your notification subscriptions directly within Mattermost
+- Notify a channel whenever something occurs on a Confluence object:
+  - Confluence spaces, including those created, updated, deleted, and restored, and those with added comments.
+  - Confluence pages, inlcuidng those created, updated, deleted, restored, and those with added, deleted, or updated comments.
```
Comment (catalintomai) [inline README.md:15]: inlcuidng ->including

**Review by catalintomai — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -5,13 +5,137 @@
...
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-confluence/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-confluence/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-A Mattermost plugin for Confluence. Supports Confluence Cloud, Server and Data Center versions. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated.  Comments and activity can be pushed into specific Mattermost channels for full visibility. 
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, as well as environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plguin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in.
```
Comment (cwarnermm) [inline README.md:10]:
```suggestion
With the Confluence plugin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in. 
```

File: README.md [hunk 5]
```diff
@@ -5,13 +5,137 @@
...
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, as well as environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plugin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in. 
 
-Installation and Usage instructions are located here: https://mattermost.gitbook.io/plugin-confluence/
+- In a Mattermost channel, setup subscriptions to get updates from Confluence and manage your notification subscriptions directly within Mattermost
+- Notify a channel whenever something occurs on a Confluence object:
+  - Confluence spaces, including those created, updated, deleted, and restored, and those with added comments.
+  - Confluence pages, inlcuidng those created, updated, deleted, restored, and those with added, deleted, or updated comments.
```
Comment (cwarnermm) [inline README.md:15]:
```suggestion
  - Confluence pages, including those created, updated, deleted, restored, and those with added, deleted, or updated comments.
```

**Comment by cwarnermm** [comment]
Thanks for the great edit, @catalintomai!

File: README.md [hunk 6]
```diff
@@ -5,13 +5,137 @@
...
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-confluence/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-confluence/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-A Mattermost plugin for Confluence. Supports Confluence Cloud, Server and Data Center versions. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated.  Comments and activity can be pushed into specific Mattermost channels for full visibility. 
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, as well as environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plugin, you can subscribe to a variety of events in Confluence and specify which channels the associated notifications will appear in.
```
Comment (cwarnermm) [inline README.md:10]:
```suggestion
With the Confluence plugin, you can subscribe to a variety of events in Confluence, and specify which channels the associated notifications will appear in. 
```

File: README.md [hunk 7]
```diff
@@ -5,13 +5,137 @@
...
 
-A Mattermost plugin for Confluence. Supports Confluence Cloud, Server and Data Center versions. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated.  Comments and activity can be pushed into specific Mattermost channels for full visibility. 
+A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data Center versions, as well as environments with multiple Confluence servers. This plugin helps your teams collaborate and keep in sync as Confluence Pages and Spaces get updated. Comments and activity can be pushed into specific Mattermost channels for full visibility. 
 
-# Documentation 
+With the Confluence plugin, you can subscribe to a variety of events in Confluence, and specify which channels the associated notifications will appear in. 
 
-Installation and Usage instructions are located here: https://mattermost.gitbook.io/plugin-confluence/
+- In a Mattermost channel, setup subscriptions to get updates from Confluence and manage your notification subscriptions directly within Mattermost
```
Comment (cwarnermm) [inline README.md:12]:
```suggestion
- In a Mattermost channel, setup subscriptions to get updates from Confluence and manage your notification subscriptions directly within Mattermost.
```

File: README.md [hunk 8]
```diff
@@ -5,13 +5,137 @@
...
-# Development 
+## Admin Guide 
+
+### Prerequisites
+
+For Confluence Plugin, Mattermost server v5.19+ is required. Confluence Server version 7.x+ is supported. Confluence Data Center is not certified.
+
+### Install the Mattermost Plugin
+
+To get started, a plugin that is installed on the Mattermost Server is needed to receive messages from the Confluence server and the notifications route it to the correct channel based on subscriptions that are configured.
```
Comment (cwarnermm) [inline README.md:25]:
```suggestion
To get started, a plugin that is installed on the Mattermost Server is needed to receive messages from the Confluence server. The notifications route it to the correct channel based on subscriptions that are configured. 
```

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**